### PR TITLE
Master dig 18

### PR DESCRIPTION
**Pull request by dj1999**

### CODE REVIEW

File: web/modules/custom/diginomica/js/disqus/disqus.js [hunk 1]
```diff
@@ -1,15 +1,27 @@
-Drupal.behaviors.diginomicaDisqusBehavior = {
-  attach: function (context, settings) {
-    var disqus_config = function () {
-      this.page.url = settings.disqus.url;
-      this.page.identifier = settings.disqus.identifier;
-    };
+(function ($, Drupal) {
```
Comment (szatom) [inline web/modules/custom/diginomica/js/disqus/disqus.js]: 1) $ (jQuery) isn't used - even it's not required as a dependency in the library definition.
2) We are not using/attaching DrupalSettings, so dependency on the library is not needed.

Comment (szatom) [reply web/modules/custom/diginomica/js/disqus/disqus.js]: Maybe once() should be added - disable multiple running on ajax call.

**Comment by szatom** [comment]
diginomica_preprocess_node

I think checking if $node is set is indeed in preprocess_node. 

typehint should be added
+    /* @var \Drupal\node\NodeInterface $node */
+    $node = $variables['node'];
+    switch ($node->bundle()) {

File: web/modules/custom/diginomica/js/disqus/disqus.js (hunk 1 above)
Comment (dj1999) [reply web/modules/custom/diginomica/js/disqus/disqus.js]: Does not fixed once() because does not work the script with once().
Preprocess node is fixed.

**Pull request merged**